aegis_control/launch/tool_cameras_driver.launch.py

```diff
@@ -0,0 +1,200 @@
+        description="Name of the wrapper node.",
+    )
+
+    declare_node_name_back_cmd = DeclareLaunchArgument(
+        "node_name_back",
+        default_value="tool_camera_back",
+        description="Name of the wrapper node.",
+    )
+
+    declare_camera_id_cmd = DeclareLaunchArgument(
+        "camera_id",
+        default_value="my_camera",
+        description="Id of the camera. Used as node namespace.",
+    )
+
+    declare_device_user_id_front_cmd = DeclareLaunchArgument(
+        "device_user_id_front",
+        default_value="basler_front",
+        description="Device user id of the camera.",
+    )
+
+    declare_device_user_id_back_cmd = DeclareLaunchArgument(
+        "device_user_id_back",
+        default_value="basler_back",
+        description="Device user id of the camera.",
+    )
+
+    declare_config_file_cmd = DeclareLaunchArgument(
+        "config_file",
+        default_value=default_config_file,
+        description="Camera parameters structured in a .yaml file.",
+    )
+
+    declare_mtu_size_cmd = DeclareLaunchArgument(
+        "mtu_size",
+        default_value="1500",
+        description="Maximum transfer unit size. To enable jumbo frames, set it to a high value (8192 recommended)",
+    )
+
+    declare_startup_user_set_cmd = DeclareLaunchArgument(
+        "startup_user_set",
+        # possible value: Default, UserSet1, UserSet2, UserSet3, CurrentSetting
+        default_value="CurrentSetting",
+        description="Specific user set defining user parameters to run the camera.",
+    )
+
+    declare_enable_status_publisher_cmd = DeclareLaunchArgument(
+        "enable_status_publisher",
+        default_value="true",
+        description="Enable/Disable the status publishing.",
+    )
+
+    declare_enable_current_params_publisher_cmd = DeclareLaunchArgument(
+        "enable_current_params_publisher",
+        default_value="true",
+        description="Enable/Disable the current parameter publishing.",
+    )
+
+    declare_respawn_cmd = DeclareLaunchArgument(
+        "respawn",
+        default_value="false",
+        description="If true, the node will be respawned if it exits.",
+    )
+
+    # Define LaunchDescription variable and return it
+    ld = LaunchDescription()
+
+    # ld.add_action(declare_node_name_cmd)
+    ld.add_action(declare_node_name_front_cmd)
+    ld.add_action(declare_node_name_back_cmd)
+
+    ld.add_action(declare_camera_id_cmd)
+
+    # ld.add_action(declare_device_user_id_cmd)
+    ld.add_action(declare_device_user_id_front_cmd)
+    ld.add_action(declare_device_user_id_back_cmd)
+
+    ld.add_action(declare_config_file_cmd)
+    ld.add_action(declare_mtu_size_cmd)
+    ld.add_action(declare_startup_user_set_cmd)
+    ld.add_action(declare_enable_status_publisher_cmd)
+    ld.add_action(declare_enable_current_params_publisher_cmd)
+
+    ld.add_action(declare_respawn_cmd)
+
+    ld.add_action(OpaqueFunction(function=_launch_node))
+
+    return ld
```

Comment:
Let's stick to the convention taken from the main UR repo (already present in other `aegis_control/launch` files):
```suggestion
    return LaunchDescription([
      # declare_node_name_cmd,
      declare_node_name_front_cmd,
      declare_node_name_back_cmd,

      declare_camera_id_cmd,

      # declare_device_user_id_cmd.
      declare_device_user_id_front_cmd,
      declare_device_user_id_back_cmd,

      declare_config_file_cmd,
      declare_mtu_size_cmd,
      declare_startup_user_set_cmd,
      declare_enable_status_publisher_cmd,
      declare_enable_current_params_publisher_cmd,

      declare_respawn_cmd,

      OpaqueFunction(function=_launch_node)])
```

Reply:
Done